stunnel/stunnel.ml

```diff
@@ -134,10 +134,34 @@ type t = {
   ; connected_time: float
   ; unique_id: int option
   ; mutable logfile: string
-  ; verified: bool
+  ; verified: config option
 }
 
-let config_file verify_cert extended_diagnosis host port =
+let appliance =
+  {
+    sni= None
+  ; verify= CheckHost
+  ; cert_bundle_path= "/etc/stunnel/xapi-stunnel-ca-bundle.pem"
+  }
+
+let pool =
+  {
+    sni= Some "pool"
+  ; verify= VerifyPeer
+  ; cert_bundle_path= "/etc/stunnel/xapi-pool-ca-bundle.pem"
+  }
```

Comment:
I feel like these definitions belong in xapi, rather than in the library code

Reply:
The library already has too much Xapi knowledge. These definitions don't preclude a client from using something else - they just provide convenient defaults for all clients. In my opinion this library should be part of xapi and not a library by itself.

Reply:
> These definitions don't preclude a client from using something else - they just provide convenient defaults for all clients.
true - if you think it's more convenient to define these here then let's just keep it as it is